src/module_2/EDA_2.md

```diff
@@ -0,0 +1,1265 @@
+    days_since_purchase_product_type    0
+    avg_days_to_buy_product_type        0
+    std_days_to_buy_product_type        0
+    dtype: int64
+
+
+
+
+```python
+# Verificar duplicados
+duplicados = df.duplicated().sum()
+print(f"Número de filas duplicadas: {duplicados}")
+
+
+```
+
+    Número de filas duplicadas: 0
+
+
+
+```python
+# Si hay duplicados, podemos eliminarlos
+df = df.drop_duplicates()
+```
+
+
+```python
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+# Separar las variables categóricas y numéricas
+variables_categoricas = ['product_type', 'vendor']
+variables_numericas = [
+    'variant_id', 'order_id', 'user_id', 'user_order_seq', 'outcome', 'ordered_before', 'abandoned_before', 
+    'active_snoozed', 'set_as_regular', 'normalised_price', 'discount_pct', 'global_popularity', 
+    'count_adults', 'count_children', 'count_babies', 'count_pets', 'people_ex_baby',
+    'days_since_purchase_variant_id', 'avg_days_to_buy_variant_id', 'std_days_to_buy_variant_id',
+    'days_since_purchase_product_type', 'avg_days_to_buy_product_type', 'std_days_to_buy_product_type'
+]
+
+# Crear gráficos para las variables categóricas
+for var in variables_categoricas:
+    plt.figure(figsize=(50, 10))
+    sns.countplot(x=var, data=df)
+    plt.title(f'Distribución de la variable categórica: {var}')
+    plt.xticks(rotation=45)  # Rotar etiquetas si es necesario
+    plt.show()
+
+# Crear gráficos para las variables numéricas
+for var in variables_numericas:
+    plt.figure(figsize=(12, 10))
+    df[var].hist(bins=30)
+    plt.title(f'Distribución de la variable numérica: {var}')
+    plt.show()
+
+```
+
+
+    
+![png](EDA_2_files/EDA_2_6_0.png)
+    
+
+
+
+    
+![png](EDA_2_files/EDA_2_6_1.png)
+    
+
+
+
+    
+![png](EDA_2_files/EDA_2_6_2.png)
+    
+
+
+
+    
+![png](EDA_2_files/EDA_2_6_3.png)
+    
+
+
+
+    
+![png](EDA_2_files/EDA_2_6_4.png)
+    
+
+
+
+    
+![png](EDA_2_files/EDA_2_6_5.png)
+    
+
+
+
+    
+![png](EDA_2_files/EDA_2_6_6.png)
+    
+
+
+
+    
+![png](EDA_2_files/EDA_2_6_7.png)
+    
+
+
+
+    
+![png](EDA_2_files/EDA_2_6_8.png)
+    
+
+
+
+    
+![png](EDA_2_files/EDA_2_6_9.png)
+    
+
+
+
+    
+![png](EDA_2_files/EDA_2_6_10.png)
+    
+
+
+
+    
+![png](EDA_2_files/EDA_2_6_11.png)
+    
+
+
+
+    
+![png](EDA_2_files/EDA_2_6_12.png)
+    
+
+
+
+    
+![png](EDA_2_files/EDA_2_6_13.png)
+    
+
+
+
+    
+![png](EDA_2_files/EDA_2_6_14.png)
+    
+
+
+
+    
+![png](EDA_2_files/EDA_2_6_15.png)
+    
+
+
+
+    
+![png](EDA_2_files/EDA_2_6_16.png)
+    
+
+
+
+    
+![png](EDA_2_files/EDA_2_6_17.png)
+    
+
+
+
+    
+![png](EDA_2_files/EDA_2_6_18.png)
+    
+
+
+
+    
+![png](EDA_2_files/EDA_2_6_19.png)
+    
+
+
+
+    
+![png](EDA_2_files/EDA_2_6_20.png)
+    
+
+
+
+    
+![png](EDA_2_files/EDA_2_6_21.png)
+    
+
+
+
+    
+![png](EDA_2_files/EDA_2_6_22.png)
+    
+
+
+
+    
+![png](EDA_2_files/EDA_2_6_23.png)
+    
+
+
+
+    
+![png](EDA_2_files/EDA_2_6_24.png)
+    
+
+
+
```

Comment:
Boolean variables make no sense to plot, other plots are too small to read and after all these plots, what are your conclusions?